packages/sommelier.rb

```diff
@@ -200,15 +199,17 @@ def self.build
             DISPLAY="${DISPLAY:0:3}"
 
             sommelier -X \
-              --x-display=${DISPLAY}  \
-              --scale=${SCALE} --glamor \
-              --drm-device=/dev/dri/renderD128 \
-              --virtwl-device=/dev/null \
-              --shm-driver=noop --data-driver=noop \
+              --x-display=${DISPLAY} \
+              --scale=${SCALE} \
+              --direct-scale \
```

Comment:
Notice the `--direct-scale` option only works on recent version of Chrome OS:
```
# on my ARM chromebook, chrome 84
sommelier.elf: ../sommelier-output.cc:180: void sl_output_init_dimensions_direct(struct sl_host_output *, int *, int *, int *, int *, int *): Assertion `host->ctx->xdg_output_manager' failed.
```

Reply:
Do you know which ChromeOS Milestone this starts working with? And is that a fatal error on the older versions?
Any ideas for a milestone version check we can use in the sommelierd script to select whether to use `--direct-scale`?

Reply:
ok I inserted code to handle this.